src/html5/node.rs

```diff
@@ -337,6 +338,65 @@ impl Node {
         namespace == MATHML_NAMESPACE
             && ["mi", "mo", "mn", "ms", "mtext"].contains(&self.name.as_str())
     }
+
+    /// Returns true if the node is an element node
+    pub fn is_element(&self) -> bool {
+        if let NodeData::Element(_) = &self.data {
+            return true;
+        }
+
+        false
```

Comment:
That's basically the `matches!` macro
```suggestion
        matches!(self.data, NodeData::Element(_))
```